src/main/java/hudson/plugins/jira/JiraRestService.java

```diff
@@ -368,4 +365,19 @@ public String getBaseApiPath() {
     public Permissions getMyPermissions() throws RestClientException {
         return jiraRestClient.getMyPermissionsRestClient().getMyPermissions(null).claim();
     }
+
+    public boolean updateIssueFieldValue(String issueKey, String fieldName, String fieldValue) {
```

Comment:
i think it should be called `updateIssue, since we update any field. and can we use this method inside current `updateIssue` (which is actually updates fix version) and `updateIssueLabels` methods ?